ui/src/pages/origination/NewConvertibleNote.tsx

```diff
@@ -117,27 +122,22 @@ const NewConvertibleNoteComponent = ({ history }: RouteComponentProps) => {
     render(el.current, data);
   }, [el, claims]);
 
-  const service = customerServices[0];
-  if (!service) return <></>;
+  if (_.isEmpty(customerServices)) return <></>;
 
   const requestOrigination = async () => {
-    const safekeepingAccount = accounts.find(
-      a => a.provider === service.payload.provider && a.id.label === account
-    );
-    if (!safekeepingAccount) {
-      console.log(
-        `Couldn't find account from provider ${service.payload.provider} with label ${account}`
-      );
+    const service = customerServices.find(i => i.payload.provider === registrar);
+    if (!service) {
+      console.log(`Couldn't find issuance service for selected registrar ${registrar}`);
```

Comment:
extra log?

Reply:
Removed - we originally had a log here where variables where `undefined` but I agree that we don't need it (see line 129)